### DISCUSSION TITLE
Different Values of Height and Width in Yolo Cfg

### DISCUSSION THREAD

**Discussion by GautamMahto** [AlexeyAB/darknet#8912] [General]
I want to ask that can we try the different values of height, width in .cfg files
Actually my training dataset is of Rectangle in shape due to this while converting into the height and width of given parameters in .cfg files it may loose some features.

**Comment by stephanecharette** [comment]
1) Note you are posting to a repo that is no longer maintained.  https://www.ccoderun.ca/programming/yolo_faq/#which_repos_to_use_and_avoid
2) See the FAQ when it comes to dimensions:  https://www.ccoderun.ca/programming/yolo_faq/#square_network
